scripts/demo/download_opencv.ps1

```diff
@@ -0,0 +1,82 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Script parameters
+param ([switch]$batch = $false)
+if ($batch) { echo "Batch mode enabled ($batch)" } else { echo "Interactive mode, use -batch for batch mode"}
+
+
+# Helper functions
+Function Batch-Exit([int]$code = 0)
+{
+    if (!$batch) { Pause }
+    Exit $code
+}
+
+# Elevate privileges if necessary
+# Note: assuming we are in <openvino>/samples/scripts, archive should be unpacked in <openvino>
+$openvino_root = [System.IO.Path]::GetFullPath("$PSScriptRoot\..\..")
+$probe_file = "$openvino_root\dummy.txt"
+try {
+    [io.file]::OpenWrite($probe_file).close()
+} catch {
+    Write-Warning "Unable to write to $openvino_root"
+    $identity = [Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()
+    if (!$identity.IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")) {
+        $opt = ""
+        if ($batch) { $opt = $opt + "-batch" }
+        Write-Warning "Launching with elevated privileges..."
+        Start-Process `
+            -FilePath powershell.exe `
+            -ArgumentList "-NoProfile -ExecutionPolicy Bypass -File `"$PSCommandPath`" $opt" `
+            -Verb RunAs
+        Batch-Exit
+    } else {
+        Write-Warning "Unable to write elevate privileges."
+        Batch-Exit 1
+    }
+}
+if([System.IO.File]::Exists($probe_file)) { rm "$probe_file" }
+
+# Default download location
+if(!$env:OPENVINO_OPENCV_DOWNLOAD_SERVER) {
+    Write-Warning "OPENVINO_OPENCV_DOWNLOAD_SERVER has not been set, using default location."
+    $url_root = "https://storage.openvinotoolkit.org/repositories/openvino/packages/master/opencv"
```

Comment:
> master

Such link would become broken in the future.

We need to specify exact version as a parameter (command-line or environment):
- use "2022.1" (OV product version) by default for storage.openvinotoolkit.org
- use internal `<package id>` with `OPENVINO_OPENCV_DOWNLOAD_SERVER`. We can still have some "latest" links with correct prefixes/suffixes (and/or "2022.1" link)